app/assets/stylesheets/nav.scss

```diff
@@ -5,15 +5,25 @@ $elegant: #e7e5e4;
 
 //************** NAVIGATION *******************************//
 .nav-container {
-  background-color: $white;
+  background: $white url('/top-bar.jpg') no-repeat center center;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background, -moz-background-size, -o-background-size, -webkit-background-size, background-size, border-bottom, box-shadow, min-height